crates/router/src/core/payment_methods/cards.rs

```diff
@@ -217,24 +221,143 @@ pub async fn create_payment_method(
 #[instrument(skip_all)]
 #[allow(clippy::too_many_arguments)]
 pub async fn create_payment_method(
```

Comment:
Payment method related function should not be in cards.rs, instead it should be in payment_method.rs file